tools/OloHeaderTool/main.cpp

```
@@ -0,0 +1,881 @@
+//
+// Scans C++ headers for OLO_PROPERTY() markers, parses the next field
+// declaration, and emits:
+//   1. C++ Mono getter/setter functions       (ScriptGlueBindings.inl)
+//   2. C++ OLO_ADD_INTERNAL_CALL registrations (ScriptGlueRegistrations.inl)
+//   3. C# proxy Component classes             (Components.Generated.cs)
+//   4. C# InternalCall declarations           (InternalCalls.Generated.cs)
+//
+// Usage:
+//   OloHeaderTool <scan_dir> <cpp_out_dir> <cs_out_dir>
+
+#include <algorithm>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)